tools/check/forbidden_strings_in_code.txt

```diff
@@ -180,3 +180,8 @@ System\.currentTimeMillis\(\)===2
 
 ### Remove extra space between the name and the description
 \* @\w+ \w+  +
+
+### Please use the MenuProvider interface now
+onCreateOptionsMenu
+onOptionsItemSelected
+onPrepareOptionsMenu
```

Comment:
This is not deprecated (yet?) on Activities, and better to avoid using a mix of solutions (this was not working well)